docs/Addons.md

```diff
@@ -1,71 +1,184 @@
+  MCT3326: 2
+
+  <your-addon-sku>: 2
+
+environment: "uhc-production"
+```
+
+For `int` and `stage`, the file will need to go into a different folder and use a different value for the `environment` key.
+
+> *NOTE*: The SKU `MCT3326` is the SKU that `osde2e` uses to provision OpenShift clusters by default. If you do not request quota of this SKU, your jobs will all fail to provision clusters.
 
 ## **Configuring OSDe2e**
 
 Once a test harness has been written, an OSDe2e test needs to be configured to install the desired add-on, then run the test harness against it. This is done by creating a PR ([example PR]) against the [openshift/release] repo. 
 
-Regarding addon testing, OSDe2e has four primary config options: `ADDON_IDS`, `ADDON_TEST_HARNESSES`, `ADDON_TEST_USER`, and `ADDON_PARAMETERS`. The first two of these are comma-delimited lists when supplied by environment variables, or YAML arrays when using the YAML config. `ADDON_IDS` informs OSDe2e which addons to install once a cluster is healthy. `ADDON_TEST_HARNESSES` is a list of addon test containers to run as pods within the test cluster. 
+Your PR will instruct Prow (our CI service) to run a pod on some schedule. You will then specify that the job should run `osde2e` with some specific environment variables and flags that describe your test harness.
+
+For addon testing, `osde2e` uses four primary environment variables: `ADDON_IDS`, `ADDON_TEST_HARNESSES`, `ADDON_TEST_USER`, and `ADDON_PARAMETERS`.
 
-`ADDON_TEST_USER` will specify the in-cluster user that the test harness containers will run as. It allows for a single wildcard (`%s`) which will automatically be evaluated as the OpenShift Project the test harness is created under.
+The first two of these are comma-delimited lists when supplied by environment variables. `ADDON_IDS` informs OSDe2e which addons to install once a cluster is healthy. `ADDON_TEST_HARNESSES` is a list of addon test containers to run as pods within the test cluster. 
+
+`ADDON_TEST_USER` will specify the in-cluster user that the test harness containers will run as. It allows for a single wildcard (`%s`) which will automatically be evaluated as the OpenShift Project (namespace) the test harness is created under.
 
 `ADDON_PARAMETERS` allows you to configure parameters that will be passed to OCM for the installation of your addon. The format is a two-level JSON object. The outer object's keys are the IDs of addons, and the inner objects are key-value pairs that will be passed to the associated addon.
 
-```
-env:
-- name: ADDON_IDS
-  value: prow-operator
-- name: ADDON_TEST_HARNESSES
-  value: quay.io/miwilson/prow-operator-test-harness
-- name: ADDON_TEST_USER
-  value: system:serviceaccount:%s:cluster-admin
-- name: ADDON_PARAMETERS
-  value: '{"prow-operator": {"foo": "bar"}}'
+An example prow job that configures the "prow" operator in the stage environment:
+
+```yaml
+- agent: kubernetes
+  cluster: build02
+  cron: 0 */12 * * *
```

Comment:
We should also call out that they are allowed/able to change the cron schedule :)